src/types/events/base-events.ts

```diff
@@ -467,6 +467,7 @@ export interface InviteRequestedEvent {
 export interface LinkSharedEvent {
   type: 'link_shared';
   channel: string;
+  is_bot_user_member: boolean;
   user: string;
   message_ts: string;
   thread_ts?: string;
```

Comment:
While verifying the actual event payload, I noticed `event_ts` is also missing.
```suggestion
  thread_ts?: string;
  event_ts: string;
```